packages/starknet-snap/src/utils/snapUtils.ts

```diff
@@ -680,3 +681,19 @@ export function toMap<k, v, z>(arr: Array<v>, key: string, keyConverter?: (v: z)
     return map;
   }, new Map<k, v>());
 }
+
+export function dappUrl(envt: string) {
+  const url = (() => {
```

Comment:
function inside a function is not require, it will generate extra memory due

Reply:
Done.